specification/network/resource-manager/Microsoft.Network/stable/2019-06-01/networkWatcher.json

```
@@ -1895,7 +1895,7 @@
           "NetworkWatchers"
         ],
         "operationId": "NetworkWatchers_GetNetworkConfigurationDiagnostic",
-        "description": "Get network configuration diagnostic.",
+        "description": "The Network Configuration Diagnostic API helps customers understand and debug network behaviour. It provides detailed information on what security rules were applied to a specified traffic flow and the result of evaluating these rules. Customers must the provide details of a flow like source, destination, protocol, etc. The API returns whether traffic was allowed or denied, the rules evaluated for the specified flow and the evaluation results.",
```

behaviour -> behavior for US spelling
```suggestion
        "description": "The Network Configuration Diagnostic API helps customers understand and debug network behavior. It provides detailed information on what security rules were applied to a specified traffic flow and the result of evaluating these rules. Customers must the provide details of a flow like source, destination, protocol, etc. The API returns whether traffic was allowed or denied, the rules evaluated for the specified flow and the evaluation results.",
```